**Primer Designing**

# Steps for the primer designing 
We are designing for the gene " KR 1020130128663-A/15" in Dunaelilla which codes for the Organic acid synthetic system using carbonic anhydrase and phosphoenolpyruvate carboxylase is as follows:- 

1.  Go to NCBI and search for the gene \"KR1020130128663-A/15\" and select the FASTA sequences among different species of Dunelilla. 

2.  Select the \"FASTA\" of each of the selected ones and copy paste it
    in word file.

3.  Now select the sequence and run the Multiple Sequence alignment on
    \"CLUSTAL W\" and run the alignment.

4.  After running we will have the result of the MSA which will look as follows :![Capture](https://github.com/prakashaman717/Prakashaman_Notebook/assets/171917083/1d8527a2-2632-46ca-bac6-e88840410515)


5.  Now, look for conserved regions within the alignment that are
    flanked by variable regions. We will design primers for it.

6.  Select the conserved regions and go the \"Primer3\" and paste your
    sequence and click on pick primers. The regions which I have
    selected the sequence are :-
    CCAGTAGCTCCTTCTTGGGGTGCAGAAGGTTTTGATCCTTACAACCCAGGTGGTGTACCTGCAGTTTTCTTTTAATGCGCCTGATGAAGACTTGCCTCAACTTACTATTGGTGGGGTTG

7.  We will get the output as following picture:- ![Picture2](https://github.com/prakashaman717/Prakashaman_Notebook/assets/171917083/40ee1d9c-307e-4654-8674-bf347241b281)

8.  Now check for the following parameters :-
Primer length: 18-25 nucleotides, Melting temperature (Tm): 50-60Â°
, CGC content: 40-60%, Avoid secondary structures (hairpins, dimers)
 We can see in the picture that the output fits the parameters so wecan proceed with it.

9.  If it fits the parameters we can quote that our primers have been designed and we can order it from the company.

    # Phylogeny Tree Construction

    Step 1: For constructing the phylogeny tree we need to download the "Mega 11."

    Step 2: Prepare Your Sequence Data

    Step 3: Click on Align

    Step 4: Select create a new alignment and copy paste your FASTA sequences that you want to align. 

    Step 5: Do alignment by ClustalW and save the file.

    Step 6: Now go the phylogeny option and open the saved file and we will get the tree.
           My tree is pasted below and the additional informations (if required) are atatched in the Git hub :- ![Assignment Bhaiya 2](https://github.com/prakashaman717/Prakashaman_Notebook/assets/171917083/c53f2ae3-4b55-44d3-9b28-4ba8a4465b6f)


